src/ingest_validation_tools/directory-schemas/merfish-v2.8.yaml

```diff
@@ -0,0 +1,159 @@
+files:
+  -
+    pattern: extras\/.*
+    required: True
+    description: Folder for general lab-specific files related to the dataset.
+  -
+    pattern: extras\/microscope_hardware\.json$
+    required: True
+    description: A file generated by the micro-meta app that contains a description of the hardware components of the microscope. Email HuBMAP Consortium Help Desk <help@hubmapconsortium.org> if help is required in generating this document.
+    is_qa_qc: True
+  -
+    pattern: extras\/microscope_settings\.json$
+    required: False
+    description: A file generated by the micro-meta app that contains a description of the settings that were used to acquire the image data. Email HuBMAP Consortium Help Desk <help@hubmapconsortium.org> if help is required in generating this document.
+    is_qa_qc: True
+  -
+    pattern: raw\/.*
+    required: True
+    description: All raw data files for the experiment.
+  -
+    pattern: raw\/additional_panels_used\.csv$
+    required: False
+    description: If multiple commercial probe panels were used, then the primary probe panel should be selected in the "oligo_probe_panel" metadata field. The additional panels must be included in this file. Each panel record should include: manufacturer, model/name, product code.
+  -
+    pattern: raw\/gene_panel\.csv$
+    required: True
+    description: The list of target genes. The expected format is: gene_id (ensembl ID), gene_name.
+  -
+    pattern: raw\/custom_probe_set\.csv$
+    required: False
+    description: 'This file should contain any custom probes used and must be included if the metadata field "is_custom_probes_used" is "Yes". The file should minimally include: target gene id, probe seq, probe id. The contents of this file are modeled after the 10x Genomics probe set file (see <https://support.10xgenomics.com/spatial-gene-expression-ffpe/probe-sets/probe-set-file-descriptions/probe-set-file-descriptions#probe_set_csv_file> ).'
```

Comment:
Does this (`raw\/custom_probe_set\.csv$`) need the quotes around it?

Reply:
Okay figured it out, I actually needed to add quotes to several other entries in order to generate docs (`:` read as a mapping character and present in 4 different descriptions).